scripts/o3de/o3de/export_project.py

```diff
@@ -1182,7 +1182,7 @@ def register_setting(key: str, description: str, default: str = None, is_passwor
 SETTINGS_OPTION_BUILD_TOOLS       = register_setting(key='option.build.tools',
                                                      description='The option to build O3DE toolchain executables. This will build AssetBundlerBatch and AssetProcessorBatch.',
                                                      is_boolean=True,
-                                                     default='False')
```

Comment:
Do SDK builds automatically ignore this variable?

Reply:
yes, that branch is completely ignored in the export scripts if you're on SDK